### DISCUSSION TITLE
Libre hosting?

### DISCUSSION THREAD

**Discussion by bocajthomas** [WeblateOrg/weblate#12037] [Q&A] [answered]
When the projects In the `trial period` is it free. While I get the projects translated all done. Because for me it says `Libre (160k strings) Trial` for 1 week

Just a little confused.

**Comment by nijel** [comment, marked as answer]
You start with a trial, and you need to ask for an approval. After the approval, it will be then free forever.

**Reply by bocajthomas** [reply]
How long does approval take?

**Reply by nijel** [reply]
It should be done within a week, but seems like @orangesunny is having some backlog now.

**Reply by bocajthomas** [reply]
Ok thank you, just hoping it will get done before expire 
Much appreciated 👏

**Reply by orangesunny** [reply]
We always make the decision before the expiration, do not worry :)

It is now approved. Happy Weblating!

**Reply by bocajthomas** [reply]
> We always make the decision before the expiration, do not worry :)
> 
> It is now approved. Happy Weblating!

Thank you ❤️
